src/main/java/com/academia/library/exception/handler/BookExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.academia.library.exception.handler;
+
+import com.academia.library.dto.ExceptionDto;
+import com.academia.library.exception.BookNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class BookExceptionHandler {
+
+    @ExceptionHandler(BookNotFoundException.class)
+    public ResponseEntity<ExceptionDto> handleBookNotFound(BookNotFoundException e) {
+        ExceptionDto message = ExceptionDto.builder()
+                .codeStatus(HttpStatus.NOT_FOUND.value())
+                .messageStatus(HttpStatus.NOT_FOUND.getReasonPhrase())
+                .messageException(e.getMessage())
+                .build();
```

Comment:
It could be great use logger like this:
log.error("Resource not found: {} ", ex.getMessage());

Reply:
Ok, thank